frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -423,6 +423,14 @@
                         vm.phases.results[i].showPrivate = true;
                     }
                 }
+                for(var i=0;i<vm.phases.results.length;i++){
+                    var timezone=moment.tz.guess();
+                    var offset=new Date(vm.phases.results[i].start_date).getTimezoneOffset();
+                    vm.phases.results[i].start_zone= moment.tz.zone(timezone).abbr(offset);
+                    offset=new Date(vm.phases.results[i].end_date).getTimezoneOffset();
+                    vm.phases.results[i].end_zone=moment.tz.zone(timezone).abbr(offset);
+                }
+                
```

Comment:
Why have another for loop? You could do the same inside above for-loop. Extract the zip and Please apply my path to this ChallenegeCtrl.js file. 
[merge_into_single_for_loop_.patch.zip](https://github.com/Cloud-CV/EvalAI/files/3161441/merge_into_single_for_loop_.patch.zip)